solution/src/sum-evens.js

```diff
@@ -0,0 +1,8 @@
+function sumEvens(numbers) {
+    if (!Array.isArray(numbers)) {
+        return undefined;
+    }
```

Comment:
If you did not guard in `hasDuplicates` why guard here?

Reply:
Just for the sake of exercising. But I'll assume input is valid (as stated in the problem description) and remove all the guards for a cleaner solution. Thank you!